crates/wasmparser/src/lib.rs

```diff
@@ -924,7 +915,7 @@ macro_rules! define_for_each_simd_operator {
         }
     };
 }
-#[cfg(feature = "simd")]
+
 _for_each_operator_group!(define_for_each_simd_operator);
```

Comment:
I think this could get dropped in theory with this change, right? The idea being that the macro would always have all the instructions?